app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+from os import mkdir, remove, rename, path
+
+
+def move_file(command: str) -> None:
+    cmd_list = command.split()
+    source = cmd_list[1]
+    destination = cmd_list[2]
```

Comment:
Consider using unpacking for `cmd_list` to improve readability and adhere to the checklist: `_, source, destination = cmd_list`.